### PR TITLE
extremely minor typo fix

### PR DESCRIPTION
**Pull request by sa-lee**
THe -> The

### CODE REVIEW

**Comment by LTLA** [comment]
Probably another dozen of these all over the place; see if you can find any of them before I merge.

**Comment by sa-lee** [comment]
Can do! I’ll have a look over the next couple of days and see if I can find anymore

**Comment by sa-lee** [comment]
are you using American spelling btw? I have found a bunch of mixed usage in clustering chapter (i.e neighbour at times, neighbor at others) just wanted to clarify before proposing fixes.

**Comment by LTLA** [comment]
Yes, I guess we should stick to American. The mixed usage is probably me fighting my instincts.

**Comment by sa-lee** [comment]
Cool updated, where I've detected it.

File: analysis/cell-cycle.Rmd [hunk 1]
```diff
@@ -232,7 +232,7 @@ For example, regression would potentially remove interesting signal if cell cycl
 We suggest only performing cell cycle adjustment on an as-needed basis in populations with clear cell cycle effects.
 
 Alternatively, users may consider just excluding cell cycle-related genes from downstream analysis.
-This should remove most of the cell cycle effect without making strong assumptions about orthogonality.
+This should remove most of the cell cycle effect without making strong assumptions about
```
Comment (LTLA) [inline analysis/cell-cycle.Rmd]: What happened here?

Comment (sa-lee) [reply analysis/cell-cycle.Rmd]: an accident. i've put it back in.

**Comment by LTLA** [comment]
Excellent, most excellent.

**Comment by sa-lee** [comment]
No probs :) thanks for the great book!

**Pull request merged**